contractcourt/commit_sweep_resolver_test.go

```diff
@@ -273,14 +274,16 @@ func TestCommitSweepResolverLocalBreach(t *testing.T) {
 	// already swept.
 	ctx.sweeper.sweepErr = sweep.ErrRemoteSpend
 
-	report := ctx.resolver.report()
-	if !reflect.DeepEqual(report, &ContractReport{
+	gotReport := ctx.resolver.report()
+	wantReport := ContractReport{
 		Outpoint:     outpoint,
 		Type:         ReportOutputUnencumbered,
 		Amount:       btcutil.Amount(amt),
 		LimboBalance: btcutil.Amount(amt),
-	}) {
-		t.Fatal("unexpected resolver report")
+	}
+	if *gotReport != wantReport {
+		t.Fatalf("unexpected resolver report. want=%+v got=%+v",
```

Comment:
Is the `+` needed?

Reply:
I fel it looks better with the `+` but I'll remove it

Reply:
Oh, I now see that `+` prints field names. Yes, that is actually better indeed. I falsely assumed there was a `String()` method. Whatever you prefer.